tests/ctst/steps/azureArchive.ts

```diff
@@ -334,6 +366,7 @@ Then('blob for object {string} fails to rehydrate',
     async function (this: Zenko, objectName: string) {
         const tarName = await isObjectRehydrated(this, objectName);
         assert(tarName);
+        await Utils.sleep(60000);
```

Comment:
Maybe worth a comment about why we set this specific value, as it slows down the tests (or some kind of check logic to reduce the duration?)